frappe/website/router.py

```diff
@@ -66,8 +66,9 @@ def get_page_context_from_template(path):
 			options = (search_path, search_path + '.html', search_path + '.md',
 				search_path + '/index.html', search_path + '/index.md')
 			for o in options:
-				if os.path.exists(o) and not os.path.isdir(o):
-					return get_page_info(o, app, app_path=app_path)
+				option = unicode(o)
```

Comment:
use `frappe.as_unicode` that will decode using utf-8, this will decode using ascii